osu.Game/Rulesets/Mods/ModAccuracyChallenge.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Configuration;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Settings;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.Judgements;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public class ModAccuracyChallenge : ModFailCondition
+    {
+        public override string Name => "Accuracy Challenge";
+        public override string Acronym => "AC";
+        public override string Description => "Fail the map if you don't maintain a certain accuracy.";
+        public override IconUsage? Icon => FontAwesome.Solid.Calculator;
+        public override ModType Type => ModType.DifficultyIncrease;
+        public override double ScoreMultiplier => 1.0;
+        public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(ModEasyWithExtraLives)).Append(typeof(ModPerfect)).ToArray();
+
+        [SettingSource("Minimum accuracy", "Trigger a failure if your accuracy goes below this value.", SettingControlType = typeof(SettingsSlider<double, PercentSlider>))]
+        public BindableNumber<double> MinimumAccuracy { get; } = new BindableDouble
+        {
+            MinValue = 0,
```

Comment:
The minimum allowed accuracy value here should probably be larger than zero, because if it is zero, then this mod does nothing useful.

Also debatable whether this mod should set `RequiresConfiguration => true`. I'd say that it should specify that.

Reply:
Should the mod also remove 100% as an accuracy value to avoid redundancy with PF?

My initial thought is that the mod shouldn't require configuration. 90% accuracy is a reasonable default for players to have to deal with. Definitely up for discussion though.

Reply:
> Should the mod also remove 100% as an accuracy value to avoid redundancy with PF?

Good question, and I don't really know what the answer is to this. Maybe this mod could supersede perfect, and use the default of 100% for backwards compatibility. Not sure, would require other opinions on this to move forward.

Reply:
I personally think that it'd be okay to have this as its own separate mod with a cap of 99% accuracy. I think most players expect mods from stable to be present in lazer, it may be a bit confusing to "remove" perfect through a supersede.

Reply:
there is no need to be "paging" anyone. all maintainers on this repository are very well aware of what is in their inboxes. a lack of response usually indicates that a particular pull is low priority (which this is).